config/config.go

```diff
@@ -43,21 +44,22 @@ func Get() (*Configuration, error) {
 	}
 
 	cfg = &Configuration{
-		BindAddr:                 ":22000",
-		KafkaAddr:                []string{"localhost:9092"},
-		AuditEventsTopic:         "audit-events",
-		GenerateDownloadsTopic:   "filter-job-submitted",
-		CodeListAPIURL:           "http://localhost:22400",
-		DatasetAPIURL:            "http://localhost:22000",
-		WebsiteURL:               "http://localhost:20000",
-		ZebedeeURL:               "http://localhost:8082",
-		ServiceAuthToken:         "FD0108EA-825D-411C-9B1D-41EF7727F465",
-		DownloadServiceSecretKey: "QB0108EZ-825D-412C-9B1D-41EF7747F462",
-		GracefulShutdownTimeout:  5 * time.Second,
-		HealthCheckInterval:      30 * time.Second,
-		EnablePrivateEnpoints:    false,
-		EnableDetachDataset:      false,
-		EnablePermissionsAuth:    false,
+		BindAddr:                    ":22000",
```

Comment:
Likewise here might be worth us alphabetically sorting these.

Reply:
See above.